boot/zephyr/Kconfig

```diff
@@ -257,9 +257,14 @@ config BOOT_SWAP_SAVE_ENCTLV
 	  JTAG/SWD or primary slot in external flash).
 	  If unsure, leave at the default value.
 
+config BOOT_ENCRYPT_IMAGE
```

Comment:
This property is used n the source code.

Reply:
I don't use it in the source code, but I use it as a helper kconfig to ensure that `MCUBOOT_VERIFY_IMG_ADDRESS` cannot be enabled if any form of image encryption is in place.

Reply:
ok, missed that